networks.json

```
@@ -1,4 +1,48 @@
 {
+    "avalancheFuji": {
+        "ERC1538": {
+            "address": "0x14B465079537655E1662F012e99EBa3863c8B9E0",
+            "startBlock": 39496460
+        },
+        "Core": {
+            "address": "0x14B465079537655E1662F012e99EBa3863c8B9E0",
+            "startBlock": 39496460
+        },
+        "AppRegistry": {
+            "address": "0x4a6531ce5150ee716b2d93865D0fbB9ce5492D17",
+            "startBlock": 39496460
+        },
+        "DatasetRegistry": {
+            "address": "0x3441A0C9FE488c51fcABa2bAAA048720f4D4F72D",
+            "startBlock": 39496460
+        },
+        "WorkerpoolRegistry": {
+            "address": "0xf81970E3CB0f8711bEaA79135e686873FfbBAC4A",
+            "startBlock": 39496460
+        }
+    },
+    "arbitrumSepolia": {
+        "ERC1538": {
+            "address": "0x14B465079537655E1662F012e99EBa3863c8B9E0",
+            "startBlock": 145249745
```

Why not the [deployment block](https://sepolia.arbiscan.io/tx/0x47b6aa0a93551becb10d00a4058969e3d6800d6cf5b7b1308cdece75328f69d5)?
```suggestion
            "startBlock": 145249746
```

---

I would like to start one block before the deployment block because I’m not sure if the start block is included. But it seems fine like this :
Fix Commit: d01ddc4b64033134f48a0fff2386612db1a32d9a